lib/transactions/sections/transaction_details_drawer.dart

```
@@ -147,7 +148,8 @@ class TransactionDetailsDrawer extends HookConsumerWidget {
                 const SizedBox(
                   width: 24,
                 ),
-                CustomTextRight(desc: transaction!.senderAddress)
+                ...createAddressWidgets(addresses: transaction!.senderAddress),
```

I made this method in order to display multiple addresses, but I didn't see them when I clicked on a transaction. I also saw a lot of mock data, so I wasn't sure where we need to remove the mock data and add in this real data. I have this here to avoid errors and still have commented code below. I'm going to clean it up, but if anyone has any suggestions for this, I'm all ears.

---

I'd make this into it's own component (can be within this file) instead of doing this in the utils file